__tests__/setup.js

```diff
@@ -0,0 +1,12 @@
+// Configuring console messages to run silently while testing with jest
+// https://medium.com/swlh/6-ways-to-run-jest-test-cases-silently-67d2fead8c11
+
+global.console = {
+  ...console,
+  log: jest.fn(),
+  debug: jest.fn(),
+};
+
+jest.spyOn(global.console, "log").mockImplementation(jest.fn());
+jest.spyOn(global.console, "warn").mockImplementation(jest.fn());
+jest.spyOn(global.console, "error").mockImplementation(jest.fn());
```

Comment:
let's keep `console.log` as-is, since it's often helpful to log things in the tests. i also don't think you need to use both of these patterns. does this still work if the contents are just:
```suggestion
// Configuring console messages to run silently while testing with jest
// https://medium.com/swlh/6-ways-to-run-jest-test-cases-silently-67d2fead8c11

global.console = {
  ...console,
  warn: jest.fn(),
  error: jest.fn(),
};
```